### PR TITLE
Allow for custom month navigation buttons

### PR DESCRIPTION
**Pull request by caseklim**
[Image from Gyazo](https://gyazo.com/80426812d12022a60639160a84e68686)

image

### CODE REVIEW

File: src/components/DateRangePicker.jsx [hunk 1]
```diff
@@ -469,8 +473,10 @@ class DateRangePicker extends React.PureComponent {
 
     const withAnyPortal = withPortal || withFullScreenPortal;
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (kevinthepan) [inline src/components/DateRangePicker.jsx]: Is this not doable in the same inline comment?

Comment (caseklim) [reply src/components/DateRangePicker.jsx]: oooh TIL https://stackoverflow.com/a/56714489

File: src/components/DateRangePicker.jsx [hunk 2]
```diff
@@ -469,8 +473,9 @@ class DateRangePicker extends React.PureComponent {
 
     const withAnyPortal = withPortal || withFullScreenPortal;
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */
```
Comment (kevinthepan) [inline src/components/DateRangePicker.jsx]: Can we move it down to where it originally was and use `eslint-disable-line`

Comment (caseklim) [reply src/components/DateRangePicker.jsx]: done 👌 I was trying to stay within the character limit but the line length is too long either way lol

**Pull request merged**